.gitignore

```diff
@@ -12,4 +12,8 @@ Publish/
 /Samples/UnrealEnginePlugin/Binaries
 /.vs
 /Brofiler/.vs/Brofiler_vs2017/v15/Server/sqlite3
-/Brofiler/.vs
\ No newline at end of file
+/Brofiler/.vs
+/Brofiler/BrofilerCore/vulkan
```

Comment:
Please remove unnecessary ignores for vulkan from the root gitignore file.
Use new version of GenerateProjects.bat file to regenerate project without gpu support.